frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -60,6 +60,7 @@ export type InspectorListItemBase = {
     search: string
     highlightColor?: 'danger' | 'warning' | 'primary'
     windowId?: string
+    windowNumber?: number | undefined
```

Comment:
Not really relevant but given you're working on windows - I'd started a PR beforehand to make windows look like tabs (with page titles instead of numbers): https://github.com/PostHog/posthog/pull/26307

It probably still needs a bunch of work so best to go with this approach for now